bindings/java/src/main/java/org/openucx/jucx/ucp/UcpContext.java

```diff
@@ -61,12 +62,62 @@ public UcpMemory registerMemory(ByteBuffer buf) {
         if (!buf.isDirect()) {
             throw new UcxException("Registered buffer must be direct.");
         }
-        return registerMemoryNative(getNativeId(), buf);
+        UcpMemory result = registerMemoryNative(getNativeId(), UcxUtils.getAddress(buf),
```

Comment:
same here, should should the last argument be `UCP_MEM_MAP_NONBLOCK`

Reply:
By default it's blocking registration.